pagefetcher.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python2
+
+import urllib
+import os
+from __builtin__ import str
+
+# set of dictionaries to store url's and names
+CISCO = {"name1": "url1", "name2": "url2"}
+TEMPLATE_TO_URL = {"cisco": CISCO}
+
+
+class UrlNotAvailable(Exception):
+    """ Exception class to raise in case of a bad URL. """
+
```

Comment:
I thought PEP8 was complaining about whitespace after the docstrings. Anyway, the coding style must comply with the one we have in `wifiphisher.py`.